qemu/tests/watchdog.py

```diff
@@ -283,8 +284,14 @@ def hotplug_unplug_watchdog_device():
                 test.fail("Can find watchdog pci")
 
         plug_watchdog_device = params.get("plug_watchdog_device", "i6300esb")
+        machine_type = params.get("machine_type")
         watchdog_device_add = ("device_add driver=%s, id=%s"
                                % (plug_watchdog_device, "watchdog"))
+        if plug_watchdog_device == "i6300esb" and machine_type == "q35":
```

Comment:
`test_type = hotplug_unplug_watchdog_device` is for `i6300esb` only. May be we can use `pcie-pci-bridge-0` directly ?

Reply:
I'm not sure that old version's qemu support 'pcie-pci-bridge' too.

Reply:
@yanan-fu @xutian  I had checked RHEL7.8 qemu version and there have pcie-pci-bridge through qemu-kvm -device ?

Reply:
Ok, let hard code bus to pcie-pci-bridge-0.